index.js

```diff
@@ -34,7 +26,7 @@ import render from './lib/Views/render';
 import GnafAddressGeocoder from 'terriajs/lib/Map/GnafAddressGeocoder.js';
 import ViewerMode from 'terriajs/lib/Models/ViewerMode.js';
 
-// Tell the OGR catalog item where to find its conversion service.  If you're not using OgrCatalogItem you can remove this.
+// Tell the OGR catalog item where to find its conversion service.
 OgrCatalogItem.conversionServiceBaseUrl = configuration.conversionServiceBaseUrl;
```

Comment:
This line is giving an error because `configuration` is undefined

Reply:
Turns out we don't need the OGR thing here any more so I've removed the reference.